geo-types/src/debug.rs

```diff
@@ -0,0 +1,466 @@
+        );
+    }
+
+    #[test]
+    fn multi_line_string_multiple_lines() {
+        let multi_line_string = wkt!(MULTILINESTRING(
+            (1 2, 3 4, 5 6),
+            (7 8, 9 10, 11 12)
+        ));
+        assert_eq!(
+            "MULTILINESTRING((1 2,3 4,5 6),(7 8,9 10,11 12))",
+            format!("{multi_line_string:?}")
+        );
+    }
+
+    #[test]
+    fn multi_line_string_multiple_lines_with_empty() {
+        let multi_line_string = wkt!(MULTILINESTRING(
+            (1 2, 3 4, 5 6),
+            EMPTY,
+            (7 8, 9 10, 11 12)
+        ));
+        assert_eq!(
+            "MULTILINESTRING((1 2,3 4,5 6),EMPTY,(7 8,9 10,11 12))",
+            format!("{multi_line_string:?}")
+        );
+    }
+    #[test]
+    fn multi_polygon_empty() {
+        let multi_polygon: MultiPolygon = wkt!(MULTIPOLYGON EMPTY);
+        assert_eq!("MULTIPOLYGON EMPTY", format!("{multi_polygon:?}"));
+    }
+
+    #[test]
+    fn multi_polygon_one_polygon() {
+        let multi_polygon = wkt!(MULTIPOLYGON(
+            ((1 2, 3 4, 5 6, 1 2))
+        ));
+        assert_eq!(
+            "MULTIPOLYGON(((1 2,3 4,5 6,1 2)))",
+            format!("{multi_polygon:?}")
+        );
+    }
+
+    #[test]
+    fn multi_polygon_multiple_polygons() {
+        let multi_polygon = wkt!(MULTIPOLYGON(
+            ((1 2, 3 4, 5 6, 1 2)),
+            ((7 8, 9 10, 11 12, 7 8))
+        ));
+        assert_eq!(
+            "MULTIPOLYGON(((1 2,3 4,5 6,1 2)),((7 8,9 10,11 12,7 8)))",
+            format!("{multi_polygon:?}")
+        );
+    }
+
+    #[test]
+    fn multi_polygon_with_holes() {
+        let multi_polygon = wkt!(MULTIPOLYGON(
+            (
+                (1 1, 10 1, 10 10, 1 10, 1 1)
+            ),
+            (
+                (20 20, 30 20, 30 30, 20 30, 20 20),
+                (22 22, 28 22, 28 28, 22 28, 22 22)
+            )
+        ));
+        assert_eq!(
+            "MULTIPOLYGON(((1 1,10 1,10 10,1 10,1 1)),((20 20,30 20,30 30,20 30,20 20),(22 22,28 22,28 28,22 28,22 22)))",
+            format!("{multi_polygon:?}")
+        );
+    }
+    #[test]
+    fn multi_polygon_with_holes_and_empty_polygon() {
+        let multi_polygon = wkt!(MULTIPOLYGON(
+            (
+                (1 1, 10 1, 10 10, 1 10, 1 1)
+            ),
+            EMPTY,
+            (
+                (20 20, 30 20, 30 30, 20 30, 20 20),
+                (22 22, 28 22, 28 28, 22 28, 22 22)
+            )
+        ));
+        assert_eq!(
+            "MULTIPOLYGON(((1 1,10 1,10 10,1 10,1 1)),EMPTY,((20 20,30 20,30 30,20 30,20 20),(22 22,28 22,28 28,22 28,22 22)))",
+            format!("{multi_polygon:?}")
+        );
+    }
+    #[test]
+    fn rect() {
+        let rect = Rect::new((1, 2), (3, 4));
+        assert_eq!("RECT(1 2,3 4)", format!("{rect:?}"));
+
+        let rect = Rect::new((3, 4), (1, 2));
+        // output is always (min, max)
+        assert_eq!("RECT(1 2,3 4)", format!("{rect:?}"));
```

Comment:
Here's an example of output that's not actually WKT, but I think it's still intuitive.